README.md

```diff
@@ -285,6 +285,9 @@ cirros | Cirros
 fedora | Fedora (amd64)
 fedora.arm64 | Fedora (arm64)
 fedora.s390x | Fedora (s390x)
+legacy | Legacy (Generic) Guest
+linux-bios | Linux (Generic) BIOS Guest
+linux-efi | Linux (Generic) EFI Guest
```

Comment:
Does `(Generic)` add that much to these names? I'd rather keep them sweet and simple.

Reply:
All right, let's keep it simple. Adjusted.

Reply:
Yeah that's fair, I also wonder if we need `linux-bios` and could get away with just `linux`?

Reply:
Done.